Jenkinsfile

```diff
@@ -72,10 +72,10 @@ pipeline {
                 }
             }
             steps {
-                withEnv(["PATH+TOOLS=$ANDROID_HOME/tools", "PATH+TOOLS_BIN=$ANDROID_HOME/tools/bin", "PATH+PLATFORM_TOOLS=$ANDROID_HOME/platform-tools"]) {
-                    sh 'scripts/delete-corrupt-avds.sh'
-                    sh 'scripts/create-avd-for-device.sh api-25-nexus-5x "system-images;android-25;google_apis;x86" "Nexus 5X" || true'
-                    sh 'scripts/create-avd-for-device.sh api-25-nexus-9 "system-images;android-25;google_apis;x86" "Nexus 9" || true'
+                script {
+                    avd.deleteCorrupt()
```

Comment:
Nice, but how does it work? Where does `avd` come from?

Reply:
![](http://www.reactiongifs.com/r/mgc.gif)

I will talk about this in our weekly.